index.js

```diff
@@ -37,6 +52,7 @@ module.exports = function serialize(obj, options) {
         options = {space: options};
     }
 
+    var uid = options.uid || UID;
```

Comment:
I'd prefer the monkey patch for the tests because I don't want people setting `options.uid` and getting pwned.